modules/virt-running-tto-pipeline-cli.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="virt-running-tto-pipeline-cli_{context}"]
+= Running the example pipelines using the CLI
+
+Use a `PipelineRun` resource to execute the example pipelines. `PipelineRun` is the running instance of a pipeline. It instantiates a pipeline for execution with specific inputs, outputs, and execution parameters on a cluster. It also creates a `TaskRun` for each task in the pipeline.
```

Comment:
I commented on the assembly file re: the use of "execute" - please refer to that comment and consider if any of these instances can be changed.